kstreams/streams.py

```diff
@@ -160,6 +160,49 @@ async def getone(self) -> ConsumerRecord:
 
         return consumer_record
 
+    async def getmany(
+        self,
+        partitions: Optional[List[TopicPartition]] = None,
+        timeout_ms: int = 0,
+        max_records: Optional[int] = None,
+    ) -> Dict[TopicPartition, List[ConsumerRecord]]:
+        """
+        Get a batch of events from the assigned TopicPartition.
+
+        Prefetched events are returned in batches by topic-partition.
+        If messages is not available in the prefetched buffer this method waits
+        `timeout_ms` milliseconds.
+
+        Attributes:
+            partitions List[TopicPartition] | None: The partitions that need
+                fetching message. If no one partition specified then all
+                subscribed partitions will be used
+            timeout_ms int | None: milliseconds spent waiting if
```

Comment:
I think the type in the comments is not needed

Reply:
It is needed, otherwise it is not shown in the docs.

<img width="798" alt="Screenshot 2023-12-14 at 08 55 09" src="https://github.com/kpn/kstreams/assets/2828842/4d554716-ff38-4dac-818d-6795ad17b058">